Engine/gfx/ali3dsw.cpp

```diff
@@ -376,6 +376,10 @@ bool ALSoftwareGraphicsDriver::Init(int virtualWidth, int virtualHeight, int rea
 
     virtualScreen = BitmapHelper::GetScreenBitmap();
 
+    // [ER] 2014-03-13
+    // Hide the system cursor via allegro
+    show_os_cursor(MOUSE_CURSOR_NONE);
```

Comment:
I mean, if this is a universal problem, perhaps there could be a better place to put this call, instead of gfx driver's Init method.

Maybe somewhere above the line where "graphics_mode_init" is called. But I cannot tell for certain, because I do not see how this works at runtime.